src/models/Preset.ts

```diff
@@ -0,0 +1,50 @@
+import { AbstractRenderer } from "../generators/AbstractRenderer";
+import { CommonInputModel } from "./CommonInputModel";
+import { CommonModel } from "./CommonModel";
+
+// TODO: Change any type to correct one
```

Comment:
🤔

Reply:
Yes, I should write also why 😄 I don't want to make big PR and in one of next I would like to fixing typing to improve DX, so at the moment I wanna to stay with it.

Reply:
Maybe just add an issue pointing to the todo, so we have it in the backlog 😄

Reply:
Added :) https://github.com/asyncapi/generator-model-sdk/issues/50